test/programs/partial-mapped-type/main.ts

```diff
@@ -0,0 +1,4 @@
+type MyObject = Partial<{
+    required: number;
+    optional: number;
```

Comment:
It's a bit confusing that both variables are optional but one has the name `required`. Why not only one?